src/styles/generic/_buttons.scss

```diff
@@ -7,9 +7,6 @@
 // =============================================================================
 
 button {
+  margin: 0 16px 0 0;
```

Comment:
I think we may want to remove this, otherwise buttons which are meant to be centered on the screen get shifted to the left.

Here's a preview with the PR enabled, notice the measure button:
![image](https://user-images.githubusercontent.com/1066253/71925990-29af1300-3147-11ea-9761-2f5bb4b061c9.png)

Here's the current button on prod:
![image](https://user-images.githubusercontent.com/1066253/71925969-1d2aba80-3147-11ea-84c2-9cc4872d6ee1.png)